mantle/kola/tests/misc/network.go

```diff
@@ -172,3 +326,147 @@ func NetworkInitramfsSecondBoot(c cluster.TestCluster) {
 		c.Fatal("networkd started in initramfs")
 	}
 }
+
+// NetworkSecondaryNics verifies that secondary NICs are created on the node
+func NetworkSecondaryNics(c cluster.TestCluster) {
+	primaryMac := "52:55:00:d1:56:00"
+	secondaryMac := "52:55:00:d1:56:01"
+	expectedPrimaryIP := "10.0.2.31"
+	expectedSecondaryIP := "10.0.2.32"
+
+	setupMultipleNetworkTest(c, primaryMac, secondaryMac)
+
+	checkSecondaryNics(c, primaryMac, secondaryMac, expectedPrimaryIP, expectedSecondaryIP)
+}
+
+func checkSecondaryNics(c cluster.TestCluster, setPrimaryMac, setSecondaryMac, expectedPrimaryIP, expectedSecondaryIP string) {
+	var err error
+	machineIdx := 1
```

Comment:
I found that the machine holding my image is in machineIdx=1, but is there a better way to check this?